web/cobrands/fixmystreet.com/layout.scss

```diff
@@ -169,6 +169,23 @@ body.fullwidthpage {
     }
 }
 
+.postcode-form-test {
+    background: transparent;
+    padding-top: 1.5em;
+}
+
+input.postcode-form-test__postcode {
+    padding: 0.5em 1em;
+    margin-top: 0.5em;
+}
+
+.postcode-form-test__buttons {
+    .btn {
+        font-size: 1.2em;
+        font-family: MuseoSans,Helmet,Freesans,sans-serif;
```

Comment:
`$body-font`?

Reply:
Hilariously, `$body-font` is defined in `sass/_base.scss`, which means it’s not available to `fixmystreet.com/layout.scss`. I guess this is why `MuseoSans,Helmet,Freesans,sans-serif` is hardcoded elsewhere in this file.

This feels like a bit of a thorny problem to solve as part of this PR. (We might want to add a `$body-font` declaration to `sass/_layout.scss`, or we might want to spin the font variables out into a `_fonts.scss` file like some of our other cobrands do.) Shall I create a new ticket for it?

Reply:
Oh, sorry!

Reply:
Actually, you know what, I’ll just add `$body-font: MuseoSans,Helmet,Freesans,sans-serif` to the top of `fixmystreet.com/layout.scss` – at least then we cut down on the duplication, and leave the door open for someone to come along and refactor the variables out in the future.